src/main/scala/org/hammerlab/guacamole/commands/jointcaller/SomaticJointCaller.scala

```diff
@@ -66,15 +65,13 @@ object SomaticJoint {
   def inputsToReadSets(sc: SparkContext,
```

Comment:
I'm confused by this comment; this is `SomaticJointCaller.scala`, `InputCollection` is only used under the `jointcaller` package atm, and this PR doesn't change that.

My change here is just updating the joint-caller's usage of shared "read set" APIs.